src/Utilities/Helpers.js

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+
+export const strong = (str) => <strong>{str}</strong>;
```

Comment:
This was done because the webpack build for cypress behaved a bit differently. In the previous implementation, there was a circular dependency which cypress webpack config wasn't able to resolve, and `strong` was evaluated to `undefined`. It's tricky and I can't say how to properly fix it, but this module separation resolves that.